modular_skyrat/modules/delam_emergency_stop/code/delam.dm

```diff
@@ -0,0 +1,50 @@
+/obj/machinery/power/supermatter_crystal
+	/// If admins and the station have been notified according to the delam suppression function
+	var/station_notified = FALSE
+
+/datum/sm_delam/proc/notify_delam_suppression(obj/machinery/power/supermatter_crystal/sm)
+	if(!sm.is_main_engine)
+		return
+
+	if(sm.station_notified)
+		return
+
+	if(world.time - SSticker.round_start_time > 30 MINUTES)
+		return
+
+	sm.station_notified = TRUE
+	log_admin("DELAM: Round timer under 30 minutes! Supermatter will perform an automatic delam suppression at strength 0%.")
+	message_admins("<font color='[COLOR_ADMIN_PINK]'>DELAM: Round timer under 30 minutes! [ADMIN_VERBOSEJMP(sm)] will perform an automatic delam suppression once integrity reaches 0%. To cancel this, press the 'Toggle Delam Suppression' verb on the EVENTS status tab.</font>")
```

Comment:
There's still no button here to do the thing from the chat itself, instead of using a verb, and you marked the previous comment about it as resolved.

Reply:
Forgot about it. Done.

![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/83487515/29c38c3f-5e94-4665-a645-815b6df2d2e6)

Reply:
@GoldenAlpharex there you go.